@dannyroberts Do you have any ideas what might be causing this error?  It's reproducible locally as well.
```
======================================================================
ERROR: setUpClass (corehq.apps.performance_sms.tests.test_schedule.TestSchedule)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/testrunner.py", line 142, in new_setUpClass
    result = original_setUpClass(*args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/testrunner.py", line 142, in new_setUpClass
    result = original_setUpClass(*args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/performance_sms/tests/test_schedule.py", line 17, in setUpClass
    delete_all_configs()
  File "/home/travis/build/dimagi/commcare-hq/corehq/util/test_utils.py", line 26, in inner
    return fn(*args, **kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/performance_sms/dbaccessors.py", line 22, in delete_all_configs
    delete_all_docs_by_doc_type(PerformanceConfiguration.get_db(), ('PerformanceConfiguration',))
  File "/home/travis/build/dimagi/commcare-hq/corehq/dbaccessors/couchapps/all_docs.py", line 49, in delete_all_docs_by_doc_type
    for chunk in chunked(get_all_docs_with_doc_types(db, doc_types), 100):
  File "submodules/dimagi-utils-src/dimagi/utils/chunked.py", line 15, in chunked
    buffer.append(it.next())
  File "/home/travis/build/dimagi/commcare-hq/corehq/dbaccessors/couchapps/all_docs.py", line 44, in get_all_docs_with_doc_types
    for result in results:
  File "/home/travis/build/dimagi/commcare-hq/corehq/util/couch_helpers.py", line 146, in paginate_view
    len_results = len(results)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/couchdbkit/client.py", line 1102, in __len__
    return self.count()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/couchdbkit/client.py", line 1025, in count
    self._fetch_if_needed()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/couchdbkit/client.py", line 1059, in _fetch_if_needed
    self.fetch()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/couchdbkit/client.py", line 1038, in fetch
    self._result_cache = self.fetch_raw().json_body
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/couchdbkit/client.py", line 1055, in fetch_raw
    return self._fetch(self._arg, self.params)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/couchdbkit/client.py", line 730, in raw_view
    return self.res.get(view_path, **params)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/restkit/resource.py", line 114, in get
    params_dict=params_dict, **params)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/couchdbkit/resource.py", line 136, in request
    response=e.response)
ResourceNotFound: missing
```

--- a/corehq/couchapps/__init__.py
+++ b/corehq/couchapps/__init__.py
@@ -5,6 +5,6 @@ CouchAppsPreindexPlugin.register('couchapps', __file__, {
     'form_question_schema': 'meta',
     'users_extra': (settings.USERS_GROUPS_DB, settings.NEW_USERS_GROUPS_DB),
     'noneulized_users': (settings.USERS_GROUPS_DB, settings.NEW_USERS_GROUPS_DB),
-    'all_docs': (None, settings.NEW_USERS_GROUPS_DB, settings.NEW_FIXTURES_DB),
+    'all_docs': (None, settings.NEW_USERS_GROUPS_DB, settings.NEW_FIXTURES_DB, 'meta'),
     'by_domain_doc_type_date': (None, settings.NEW_USERS_GROUPS_DB, settings.NEW_FIXTURES_DB, 'meta'),
 })

--- a/corehq/couchapps/tests/test_all_docs.py
+++ b/corehq/couchapps/tests/test_all_docs.py
@@ -31,6 +31,7 @@ class AllDocsTest(TestCase):
              get_all_doc_ids_for_domain_grouped_by_db(self.domain)},
             {get_db(None).uri: ['main_db_doc'],
              get_db('users').uri: ['users_db_doc'],
+             get_db('meta').uri: [],
              get_db('fixtures').uri: []}
         )
 

